### PR TITLE
New adapter: Colombia

### PR DESCRIPTION
**Pull request by ColombiaOnline**
## Type of change

- [x] new adapter

## Description of change

 We have uses some conditions for banner sizes.

- contact email of the adapter’s maintainer
 anil.singh1@timesinternet.in

### CODE REVIEW

**Comment by ColombiaOnline** [comment]
@musikele ,
Thanks for suggestions. 
We have resolved all issues. please commit changes in prebid repository.

**Comment by ColombiaOnline** [comment]
@musikele ,
Thanks for response.
We have resolved the issue and execute gulp test with lint and getting 0 errors. please commit changes in prebid repository.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "prebid.js",
-  "version": "3.12.0",
+  "version": "3.17.0-pre",
```
[the remark this answers is not in the document]
Comment (ColombiaOnline) [reply package-lock.json]: Yes, I have removed these files and updated

File: test/spec/modules/colombiaBidAdapter_spec.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        'sizes': [
+          [300, 250]
+        ],
+        'bidId': '382091349b149f"',
+        'bidderRequestId': '"1f9c98192de251"',
+        'auctionId': '61466567-d482-4a16-96f0-fe5f25ffbdf1',
+      }
+    ];
+
+    const request = spec.buildRequests(bidRequests);
```
[the remark this answers is not in the document]
Comment (ColombiaOnline) [reply test/spec/modules/colombiaBidAdapter_spec.js:72]: I have resolved the issues and getting 0 errors after running gulp test.

**Comment by musikele** [comment]
I think you misunderstood my last comment; i didn't ask to delete the files from the repository, because if I merge your PR, package.json and package-lock.json will be deleted too. 

I asked to delete the files from the PR, so that only the two files related to your bidder will be included. 

I suggest you to: 

1. close this PR 
2. update your ColombiaOnline repository to the latest prebid commit
3. create a NEW pull request from your repository to Prebid 
4. add only the files colombiaBidAdapter.md, colombiaBidAdapter.js, colombiaBidAdapter_spec.js to the commit. 
5. tag me in the comments of the PR so I'll review it asap. 

If you need any help in any of those points, ping me.

**Pull request closed without merging**